dvc/cli.py

```diff
@@ -60,6 +61,7 @@
     metrics,
     install,
     root,
+    listCmd,
```

Comment:
A bit weird that it is `listCmd` here, but `list_outs.py` in `dvc/repo/`. Let's use something in particular. Thinking about `ls` again now, but not sure :slightly_smiling_face:

Reply:
Yes, that just happened
I'm going to rename `listCmd` to `list_outs`

Maybe should I use it as `command.list`? But it becomes inconsistency

> could use dvc ls and use ls

+1

Reply:
Or we might use `ls` for `ls.py` and API method, but keep `dvc list` in CLI, if there are any objections about `dvc list`. Same as we have `dvc import` but `imp()` and `imp.py`.